decidim/decidim
# Load the editor assets only when there is an editor on the page

#### :tophat: What? Why?
It was discussed before that we should aim to optimize the amount of JS and CSS loaded on the pages.

This change will offload the editor assets (JS/CSS) only to those pages where we display editors, so that we are not loading these assets when they are not really needed.

#### :pushpin: Related Issues
- Related to
  * https://github.com/decidim/decidim/issues/10090#issuecomment-1327187674
  * #8550

#### Testing
1. Go to the front page of Decidim
2. Inspect the source
3. See that there are no editor assets
4. Go to the admin panel to a view which has an editor
5. Inspect the source
6. See that the editor assets are loaded correctly
